terraform/postgresql/provision/variables.tf

```diff
@@ -20,8 +20,9 @@ variable "storage_gb" { type = number }
 variable "publicly_accessible" { type = bool }
 variable "multi_az" { type = bool }
 variable "instance_class" { type = string }
-variable "engine" { type = string }
-variable "engine_version" { type = string }
+variable "postgres_version" { type = string }
+#variable "postgres_version_major" { type = string }
```

Comment:
*question:* Why do we want to keep these variables commented out?

Reply:
Nope- left there accidentaly

Reply:
Fixed it

Reply:
I think pushing that change wasn't successful.